web/template/dashboard.gotmpl

```diff
@@ -0,0 +1,45 @@
+{{ template "page" . }}
+
+{{ define "title" }}Your cases{{ end }}
+
+{{ define "main" }}
+  <h1 class="govuk-heading-xl">Your cases</h1>
+
+  <table class="govuk-table">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Donor</th>
+        <th scope="col" class="govuk-table__header">Case</th>
+        <th scope="col" class="govuk-table__header">LPA type</th>
+        <th scope="col" class="govuk-table__header">Received</th>
+        <th scope="col" class="govuk-table__header">Worked</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      {{ range .Cases }}
+        <tr class="govuk-table__row">
+          <th scope="row" class="govuk-table__header">{{ .Donor.DisplayName }}</th>
+          <td class="govuk-table__cell">
+            <a href="{{ sirius (printf "/lpa/person/%d/%d" .Donor.ID .ID) }}" class="govuk-link">
+              {{ .Uid }}
+            </a>
+          </td>
+          <td class="govuk-table__cell">
+            {{ upper .SubType }}
+          </td>
+          <td class="govuk-table__cell">
+            {{ format .ReceiptDate "02 Jan 2006" }}
```

Comment:
I assume we are going to always format dates the same way, so maybe just hard code the format?

Reply:
Yep, and renamed to `formatDate` because I remember I hate things generically named "format"